### ISSUE TITLE
New option in display menu

### ISSUE THREAD

**Issue by IDknoobcoder** [spacehuhn/esp8266_deauther#1746]
Hi guys I wanted to add new display menu of ir that blasts ir codes of power off could anyone reading help me with this

**Comment by ihacksemant** [comment]
> Hi guys I wanted to add new display menu of ir that blasts ir codes of power off could anyone reading help me with this

hii, I'm interested in making a flipper zero like tool too, if we are at it already why not also try adding the famous 433 mhz rf module too

**Comment by IDknoobcoder** [comment]
Yeah we could also add cc1101 module it is also cheap wire it with esp but main problem is code also failed adding an ir menu to display so to turn off devices without remote using universal code tell if u could help

**Comment by IDknoobcoder** [comment]
Also I am assembling it in a gshock I think cc1101 would be big for it but anyways if you could help then plsss🙏🏻🙏🏻
